--- a/third_party/wallpaper-engine-renderer/src/backend/web/CMakeLists.txt
+++ b/third_party/wallpaper-engine-renderer/src/backend/web/CMakeLists.txt
@@ -67,7 +67,8 @@
         libcef_dll_wrapper)
     target_include_directories(we-cef-helper PRIVATE
         ../..
-        ../../../include)
+        ../../../include
+        ${CEF_ROOT})
     target_compile_options(we-cef-helper PRIVATE ${warn_opts} -fno-exceptions)
     target_link_options(we-cef-helper PRIVATE
         -Wl,-e,_start
